jenkins/util.groovy

```diff
@@ -99,3 +99,23 @@ void failedJiraTask(String projkey, String recipe, String machine) {
                           issuetype: [name: 'Task']]]
    def newIssue = jiraNewIssue issue: issue, site: 'JIRA_SITE'
 }
+
+/**
+* Create a Jira Service Desk ticket with custom message
+*
+* @param title Subject of the Jira Service Desk ticket appended to [${machine}].
+* @param machine Computing system where the build took place.
+* @param message Content of the Jira Service Desk ticket prepended to Jenkins job details.
+* @param queue Queue where the Jira Service Desk ticket will be dispatched.
+*/
+void createJiraSD(String title, String machine, String message, String queue) {
+
+   def title = "[${machine}] ${title}"
```

Comment:
Remove this and put it directly in the `summary` below because there is already a `title` defined.

Reply:
Thanks @teojgo, I've changed the parameter name to `subject`, in order to avoid conflicts